starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -809,16 +810,13 @@ sub large_dither_checks {
     my $self = shift;
     my $dither_state = shift;
     my $all_dither = shift;
-    my $time_tol = 10;         # Commands must be within $time_tol of expectation
+    my $time_tol = 11;         # Commands must be within $time_tol of expectation
```

Comment:
Increased time tolerance by 1s.  My impression for these big dither tests is that a change in the tolerance from 10 to 11 (or possible 21 when including extra 10 second pad ) is fine.

Reply:
Agreed.

Reply:
The large dither checks are somewhat broken anyway, because we have the defect in flight starcheck that the operational process has changed (disable dither to start not required) and the code has not caught up.